Documentation/network/bgp-control-plane/bgp-control-plane.rst

```diff
@@ -15,6 +15,11 @@ from outside the cluster for environments that support BGP. Because BGP
 Control Plane does not program the :ref:`datapath <ebpf_datapath>`, do not use it to establish
 reachability within the cluster.
 
+.. admonition:: Video
+  :class: attention
+
+  For more insights on Cilium's BGP, check out `eCHO Episode 101: More BGP fun with Cilium  <https://www.youtube.com/watch?v=Tv0R6VxyWhc>`__.
```

Comment:
I don't remember the list of all episodes, but “more BGP fun” implies there's a former episode dealing with BGP as well? If that's the case, would it be worth linking the two episodes here?

Reply:
not really, there's no episode leading to this actually, It's just the title that might be confusing